api/endpoints.go

```diff
@@ -380,6 +419,23 @@ func handleSystem(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		existingPlaytime, err := db.RetrievePlaytime(uuid)
+		if err != nil && !errors.Is(err, sql.ErrNoRows) {
+			httpError(w, r, fmt.Errorf("failed to retrieve playtime: %s", err), http.StatusInternalServerError)
+			return
+		} else {
+			playtime, ok := system.GameStats.(map[string]interface{})["playTime"].(float64)
+			if !ok {
+				httpError(w, r, fmt.Errorf("no playtime found"), http.StatusBadRequest)
+				return
+			}
+
+			if float64(existingPlaytime) > playtime {
+				httpError(w, r, fmt.Errorf("session out of date: existing playtime is greater"), http.StatusBadRequest)
+				return
+			}
```

Comment:
and here

Reply:
Everything outside of the checks is expected behavior.